src/components/pages/datasets/cards/requests.tsx

```diff
@@ -1,40 +1,66 @@
 import type {FunctionComponent, MouseEventHandler} from 'react'
-import {Card, ButtonAsLink} from '@/components/lib'
+import {Card, ButtonAsLink, ButtonAsIcon} from '@/components/lib'
+import {Check} from '@/components/icons/check'
+import {Close} from '@/components/icons/close'
+import {User} from '@/components/icons/user'
 
-// TODO: update types all over this file
 // TODO: check permissions with Ionesio
 // TODO: check budget logic with Ionesio
 
-const UserRequestingUI: FunctionComponent<{
-  userAvatarURL: string
-  userName: string
-  userId: string | number
+interface PermissionRequestProperties {
+  user_email: string
```

Comment:
Haven't sticked to camelCase because I was mimicking the api response fields to match the interface. As I said earlier, the best workaround might be to use the global types for requests and then extend those to add custom functionality to the component ( if needed )

Reply:
We've added a "camelizer" to each axios response (and a decamelize to
requests). So everything that comes from the API via the axios call
is going to be camelCased.